app/src/main/java/edu/grinnell/appdev/grinnelldirectory/Activity/LoginActivity.java

```diff
@@ -70,7 +69,7 @@ void signIn(View view) {
             Intent intent = new Intent(getBaseContext(), MainActivity.class);
```

Comment:
`final`ize the intent. 

Also, you should pass in `this` as an argument to the `Intent` instance.  `LoginActivity` is, in itself, a `Context` by virtue of extending `AppCompactActivity` (this class extends `FragmentActivity` which extends `Activity` which, by a chain of inheritance from `ContextThemeWrapper` and `ContextWrapper` extends `Context`).